apps/rush-lib/src/cli/actions/VersionAction.ts

```diff
@@ -221,7 +221,7 @@ export class VersionAction extends BaseRushAction {
     });
 
     if (packageJsonUpdated) {
-      git.addChanges();
+      git.addChanges(':/*');
```

Comment:
@octogonz Will `git add :/**/package.json` make more sense here?

Reply:
Something like `git.addChanges(':/**/' + FileConstants.PackageJson);`

Reply:
Sure. This is unrelated to #669 though, right? That issue was just about the cwd being incorrect when Git is invoked

Reply:
This seems related. Currently, is what I get when I run the repro command:

...
warning: LF will be replaced by CRLF in webpack/set-webpack-public-path-plugin/CHANGELOG.json.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in webpack/set-webpack-public-path-plugin/CHANGELOG.md.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in webpack/set-webpack-public-path-plugin/package.json.
The file will have its original line endings in your working directory

* DRYRUN: `git add . (..\..\changes)`

* DRYRUN: `git add **/CHANGELOG.json`

* DRYRUN: `git add **/CHANGELOG.md`

* DRYRUN: git commit -m Deleting change files and updating change logs for package updates.

* DRYRUN: `git add .`

* DRYRUN: git commit -m Applying package updates.

* DRYRUN: git push origin HEAD:version/bump-1562955216579 --follow-tags --verbose --no-verify

* DRYRUN: git fetch origin

* DRYRUN: git checkout undefined

* DRYRUN: git pull origin undefined

* DRYRUN: git merge version/bump-1562955216579 --no-edit

* DRYRUN: git push origin HEAD:undefined --follow-tags --verbose --no-verify

* DRYRUN: git branch -d version/bump-1562955216579

* DRYRUN: git push origin --delete version/bump-1562955216579



I thought I will fix all the highlighted lines. Thoughts?

Reply:
According to what I’ve read, this should work and fix #669 

Did you get all the places git should use the base of the repo?

Reply:
I got all the places for `rush version` and `rush publish`, yes. Thanks @halfnibble.

Reply:
@octogonz Did you want Sachin to update this PR to use `git.addChanges(':/**/' + FileConstants.PackageJson);`? Or is it fine as is?